e2e/api_test.go

```diff
@@ -1,131 +1,73 @@
 package main
 
 import (
-	"bytes"
 	"context"
-	"io/ioutil"
-	"net/http"
 	"os"
 	"path/filepath"
 	"testing"
 
 	"github.com/cosmos/cosmos-sdk/crypto/keys"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/types/rest"
 	"github.com/mesg-foundation/engine/app"
 	"github.com/mesg-foundation/engine/config"
 	"github.com/mesg-foundation/engine/cosmos"
 	pb "github.com/mesg-foundation/engine/protobuf/api"
 	"github.com/stretchr/testify/require"
-	rpcclient "github.com/tendermint/tendermint/rpc/client"
 	"google.golang.org/grpc"
 )
 
 type apiclient struct {
-	pb.ServiceClient
 	pb.EventClient
-	pb.ExecutionClient
-	pb.ProcessClient
-	pb.InstanceClient
-	pb.OwnershipClient
 	pb.RunnerClient
 }
 
 var (
 	minExecutionPrice     sdk.Coins
 	client                apiclient
-	cclient               *cosmos.Client
 	cdc                   = app.MakeCodec()
 	processInitialBalance = sdk.NewCoins(sdk.NewInt64Coin("atto", 10000000))
+	kb                    *cosmos.Keybase
+	cfg                   *config.Config
 	engineAddress         sdk.AccAddress
 )
 
-const (
-	lcdEndpoint        = "http://127.0.0.1:1317/"
-	lcdPostContentType = "application/json"
-)
-
-func lcdGet(t *testing.T, path string, ptr interface{}) {
```

Comment:
why deleting these functions, they were good and with the asserts that are missing in the new one

Reply:
fixed in https://github.com/mesg-foundation/engine/pull/1758